platform/core/src/services/DicomMetadataStore/createStudyMetadata.js

```diff
@@ -28,10 +29,12 @@ function createStudyMetadata(StudyInstanceUID) {
      *
      * @param {object[]} instances
      * @param {string} instances[].SeriesInstanceUID
+     * @param {string} instances[].StudyDescription
      * @returns {bool} true if series were added; false if series already exist
      */
     addInstancesToSeries: function(instances) {
       const { SeriesInstanceUID } = instances[0];
+      if (!this.StudyDescription) this.StudyDescription = instances[0].StudyDescription;
```

Comment:
{}

Reply:
I really dislike making the code less readable for the sake of formal style rules, but whatever.  A single line keeping the code shorter makes it easier to read because it is obvious that line is a single unit.